tests/commands/test_commands.py

```diff
@@ -808,6 +808,7 @@ def test_download_data_trades(mocker, caplog):
         "--exchange", "kraken",
         "--pairs", "ETH/BTC", "XRP/BTC",
         "--days", "20",
+        "--trading-mode", "spot",
```

Comment:
I'm not sure that's necessary (or even desired).

Default for this is spot - and we should assume that users will not provide the argument if it ain't required.